feature/gribi/ate_tests/base_leader_election_test/base_leader_election_test.go

```diff
@@ -247,10 +246,9 @@ func testIPv4LeaderActiveChange(ctx context.Context, t *testing.T, args *testArg
 	args.clientA.AddIPv4(t, ateDstNetCIDR, nhgIndex+2, *deviations.DefaultNetworkInstance, "", fluent.InstalledInRIB)
 
 	// Verify the entry for 198.51.100.0/24 is active through AFT Telemetry.
+	t.Log("Verify the entry for 198.51.100.0/24 is active through AFT Telemetry.")
```

Comment:
Please use the var ateDstNetCIDR and avoid using hard-coded IP address to increase maintainability.